src/SurveyService.Models.Dto/Requests/Group/CreateGroupRequest.cs

```diff
@@ -0,0 +1,15 @@
+using LT.DigitalOffice.SurveyService.Models.Dto.Requests.Question;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace LT.DigitalOffice.SurveyService.Models.Dto.Requests.Group;
+
+public record CreateGroupRequest
+{
+  [Required(ErrorMessage = "Subject is required.")]
+  public string Subject { get; set; }
+  public string Description { get; set; }
+
+  [Required(ErrorMessage = "At least 1 question is required.")]
```

Comment:
Are you sure that an empty list won't pass validation?

Reply:
Done